edge-modules/SimulatedTemperatureSensor/docker/windows/arm32v7/base/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Base image for arm32v7 dotnet-runtime container
+FROM mcr.microsoft.com/windows/nanoserver/insider:10.0.17763.55
+ 
+ENV DOTNET_VERSION 2.1.5
+ENV DOTNET_DOWNLOAD_URL https://dotnetcli.blob.core.windows.net/dotnet/Runtime/$DOTNET_VERSION/dotnet-runtime-${DOTNET_VERSION}-win-arm.zip
+ 
+RUN curl -SL %DOTNET_DOWNLOAD_URL% --output dotnet.zip \
+    && mkdir "c:/program files/dotnet" \
+    && tar -zxf dotnet.zip -C "c:/program files/dotnet" \
+    && del dotnet.zip
+ 
+USER ContainerAdministrator
+# In order to set system PATH, ContainerAdministrator must be used
+RUN setx /M PATH "%PATH%;C:\Program Files\dotnet"
+# Add an unprivileged user account for running the module
+RUN net user /add moduleuser
```

Comment:
Are `moduleuser` and `ContainerUser` (next line) supposed to be two different accounts, or the same account?

Reply:
moduleuser is the user we created for running module with non-privilege permission.  AFAIK ContainerUser is the user created by Docker; which has non-root privilege.  I think it is better to use moduleuser.